### DISCUSSION TITLE
When will Node 22 be supported.

### DISCUSSION THREAD

**Discussion by DarraghOFlaherty** [Azure/static-web-apps#1557] [General]
Node 20 has entered maintenance mode with node 22 being the current active LTS version. When will Azure Static Web Apps, support node 20?

**Comment by owljacob** [comment]
https://github.com/Azure/azure-functions-ux/pull/7841

**Comment by mheob** [comment]
[Node.js 22 is LTS](https://nodejs.org/en/about/previous-releases) now.
When will Azure Static Web Apps support it?

**Comment by luckyabsoluter** [comment]
https://github.com/Azure/static-web-apps/issues/1534
